policy/modules/kernel/kernel.if

```diff
@@ -4283,3 +4284,23 @@ interface(`kernel_unlabeled_entry_type',`
 	allow $1 unlabeled_t:file { mmap_exec_file_perms ioctl lock };
 ')
 
+
+########################################
+## <summary>
+##	Allow the caller load a new kernel
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`kernel_kexec_load',`
+	gen_require(`
+		type unlabeled_t;
+	')
```

Comment:
Forgot to remove this?

Reply:
A result of rebasing when the block appeared back.